diff --git a/security/Makefile b/security/Makefile
index 4601230ba..194234213 100644
--- a/security/Makefile
+++ b/security/Makefile
@@ -26,6 +26,7 @@ obj-$(CONFIG_CGROUPS)			+= device_cgroup.o
 obj-$(CONFIG_BPF_LSM)			+= bpf/
 obj-$(CONFIG_SECURITY_LANDLOCK)		+= landlock/
 obj-$(CONFIG_SECURITY_IPE)		+= ipe/
+obj-$(CONFIG_SECURITY_AEGIS)		+= aegis/
 
 # Object integrity file lists
 obj-$(CONFIG_INTEGRITY)			+= integrity/
